tests/addresses_events/addresses_events_ethereum_first_funded_by_assert_unique.sql

```diff
@@ -0,0 +1,6 @@
+-- Check for multiple addresses
+
+SELECT address
```

Comment:
you don't need this test at all, you can simply apply the built-in unique test on the schema file for this model (tests: -unique) under the column ```address```